spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/SparkSQLProperties.java

```diff
@@ -90,4 +90,8 @@ private SparkSQLProperties() {}
   public static final String EXECUTOR_CACHE_LOCALITY_ENABLED =
       "spark.sql.iceberg.executor-cache.locality.enabled";
   public static final boolean EXECUTOR_CACHE_LOCALITY_ENABLED_DEFAULT = false;
+
+  // Controls whether column statistics are enabled when estimating statistics
```

Comment:
the comment seems not meaningful at first read.  Can we be more precise?

Reply:
Changed to `// Controls whether to calculate column statistics and report them to Spark`